PF/Assignment1/ConcordanceOfCharacters.java

```diff
@@ -0,0 +1,59 @@
+//Program to find occurrence of elements in a string
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.Set;
+
+public class ConcordanceOfCharacters {
+
+	int k = 0;
+	Set<Character> setA = new HashSet<Character>();
+	String input;
+	int length;
+	String storeIndex;
+	String result[];
+
+	// constructor to initialize input string and its length
+	public ConcordanceOfCharacters(String input, int length) {
+		this.input = input;
+		this.length = length;
+		result = new String[length];
```

Comment:
ACTION: If you assign result array length to the length of input string then there will be lost of memory wastage if all characters are not unique in input string. It is only good for worst case but for best cases it will always consume more memory, so it should be equivalent to length of unique characters.